jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -196,7 +196,7 @@ class DeploySamApplicationValidator {
         return null
     }
 
-    private fun validateStackName(name: String?): String? {
+    private fun validateStackName(name: String?, existingNames: List<String>): String? {
```

Comment:
if we are going to leave this as a parameter it should be a `Set` rather than a list - it's got a more efficient `exists` runtime complexity.

Reply:
This parameter is removed now.